src/geographic_supplemental_attribute.jl

```diff
@@ -0,0 +1,33 @@
+"""
+Base type for structs that store infos
+
+Required interface functions for subtypes:
+
+  - get_internal()
+  - get_components_uuid()
+  - get_time_series_container()
+
+Subtypes may contain time series, if no time series container is implemented return nothing
+"""
+struct InfrastructureSystemsGeo <: InfrastructureSystemsSupplementalAttribute
```

Comment:
How about `GeographyCoordinates`? Do we need an abstract type for geography-related structs to allow for different data models?

This struct will be exported into PSY, right? If so we probably don’t want `InfrastructureSystems` in the name.

Reply:
Agree con changing the name but in principle this can store the definition of a polygon or a line so it is more than coordinates